lib/auth/person.ex

```diff
@@ -214,6 +230,8 @@ defmodule Auth.Person do
   def get_person_by_email(email) do
     __MODULE__
     |> Repo.get_by(email_hash: email)
+    |> Repo.preload(:roles)
+    |> Repo.preload(:statuses)
```

Comment:
```suggestion
    |> Repo.preload([:statuses, :roles])
```

Would reduce passes on the database